src/main/java/duke/command/AddEventCommand.java

```diff
@@ -0,0 +1,46 @@
+package duke.command;
+
+import duke.util.Parser;
+import duke.util.TaskList;
+import duke.task.Event;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents the Command of adding a new event to a task list.
+ */
+public class AddEventCommand extends Command {
```

Comment:
Should class names be nouns? Maybe can rename to something like `EventCommand` here?